### PR TITLE
Task#223038:[WEB] -exam attendance module - Change in radio.group to input type radio

### PR DESCRIPTION
**Pull request by snehalsabade**
https://tracker.tekdi.net/issues/223038

### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker

## Summary by CodeRabbit

- **Refactor**
    - Updated various components to use custom radio button implementations with controlled state for better encapsulation and control flow.

- **Accessibility**
    - Enhanced accessibility by replacing `Radio.Group` components with `div` and `label` structures in multiple files.

- **Functionality**
    - Improved state management and event handling in `StatusButton`, `MarkLearnerAttendance`, `ExamSchedule`, `ScheduleExam`, and `ExamResult` components.
    - Added safety checks using optional chaining in `CustomAccordion`.

### CODE REVIEW

File: apps/front-end/src/pages/admin/facilitator/view/StatusButton.js [hunk 1]
```diff
@@ -15,20 +15,30 @@ import { useNavigate } from "react-router-dom";
 
 const CRadio = ({ items, onChange }) => {
   const { t } = useTranslation();
+  const [selectedValue, setSelectedValue] = useState("1");
+
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/admin/facilitator/view/StatusButton.js:19]: resolved

File: apps/front-end/src/pages/volunteer-admin/volunteer/view/StatusButton.js [hunk 2]
```diff
@@ -15,20 +15,30 @@ import { useNavigate } from "react-router-dom";
 
 const CRadio = ({ items, onChange }) => {
   const { t } = useTranslation();
+  const [selectedValue, setSelectedValue] = useState("1");
+
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/volunteer-admin/volunteer/view/StatusButton.js:19]: resolved

File: apps/front-end/src/pages/volunteer-admin/volunteer/view/StatusButton.js [hunk 3]
```diff
@@ -15,20 +15,30 @@ import { useNavigate } from "react-router-dom";
...
+  const handleChange = (event) => {
+    setSelectedValue(event.target.value);
+    onChange(event);
+  };
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/volunteer-admin/volunteer/view/StatusButton.js:23]: not required

File: apps/front-end/src/pages/admin/facilitator/view/Shortlisted.js [hunk 4]
```diff
@@ -35,16 +35,20 @@ function Shortlisted({ status, setStatus, update, data }) {
...
-              <Radio value="2" size="sm">
+              </label>
+              <label>
+                <input type="radio" name="radioGroup" value="2" size="sm" />
                 {t("NO")}
-              </Radio>
+              </label>
             </HStack>
-          </Radio.Group>
+          </div>
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/admin/facilitator/view/Shortlisted.js:50]: not requird

File: apps/front-end/src/v2/components/Functional/ExamResult/ExamResult.js [hunk 5]
```diff
@@ -122,15 +123,16 @@ const ExamResult = ({ userTokenInfo, footerLinks }) => {
...
+              <label key={board.id}>
+                <input
+                  colorScheme="red"
+                  type="radio"
+                  value={board.id}
+                  checked={selectedId === board.id.toString()}
+                  onChange={handleSelect}
+                />
+                {board.name}
+              </label>
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/v2/components/Functional/ExamResult/ExamResult.js:135]: not requird

File: apps/front-end/src/pages/admin/PoAdmin/Exam/ScheduleExam.js [hunk 6]
```diff
@@ -333,15 +334,20 @@ function ScheduleExam() {
...
+                    value={filter?.board_id || null}
+                    key={board?.board?.id}
+                    onChange={(board) => {
+                      handleSelect(board);
+                      setMaxDate(board?.addedMaxDate);
+                      setSelectedBoardId(board);
+                    }}
+                  />
+                  {board?.board?.name}
+                </label>
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/admin/PoAdmin/Exam/ScheduleExam.js:350]: not requird

**Review by swapnilphalke03 — APPROVED** [review]

**Pull request merged**